framework/src/main/java/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -845,6 +922,16 @@ public Boolean visitTypevar_Typevar(
       }
     }
 
+    if (TypesUtils.isCapturedTypeVariable(subtype.getUnderlyingType())
+        && TypesUtils.isCapturedTypeVariable(supertype.getUnderlyingType())) {
```

Comment:
Are these necessarily the *same* captured type variable, or might they be different ones?